angular-app/proxy.conf.json

```diff
@@ -1,6 +1,8 @@
 {
-  "/api": {
+  "disabled": {
```

Comment:
Shouldn't this be kept to `/api`? otherwise the dev proxy won't work.
If it's not needed anymore, the entire section can be removed